### DISCUSSION TITLE
About the config maxRoleNum&maxUserNum

### DISCUSSION THREAD

**Discussion by bingbb47** [milvus-io/milvus#24799] [Q&A and General discussion] [answered]
![image](https://github.com/milvus-io/milvus/assets/111261344/252b2d48-b039-4e41-a4cf-6334c9a29272)
We can see that, the max role num we can create is 10 and the max user num is 100 in the defalut config.
Ofcourse, i can configure the value by 
![image](https://github.com/milvus-io/milvus/assets/111261344/09dcd49f-793d-4f76-ad6a-caed52d383ef)

I want to know , why the default value are so small and will something wrong if i turn these value too large?

**Comment by yhmo** [comment, marked as answer]
You can increase the maxUserNum/maxRoleNum to a large value. Just keep in mind that a large number of user/role will bring high pressure to Etcd service.

**Reply by bingbb47** [reply]
thanks, get it

**Reply by ganderaj** [reply]
Can you please comment on how do we update our existing Milvus-EKS (v2.3.x) implementation to reflect changes ?

**Reply by yhmo** [reply]
@ganderaj 
https://milvus.io/docs/configure-helm.md#Configure-Milvus-via-configuration-file
https://milvus.io/docs/configure_operator.md
